### ISSUE TITLE
Feature: Add CSS and HTML linting using Github Actions

### ISSUE THREAD

**Issue by kdipippo** [dscabesit/playactivity#39]
Right now, the files in the repo have inconsistent formatting issues. There's extraneous whitespace errors. This could also be elaborated to focus on HTML accessibility, i.e. https://www.a11yproject.com/.

Github Actions should be incorporated so that the linter runs every time a commit is pushed to a PR. This would enforce these standards constantly and consistently so that only polished code exists on the `main` branch.

I was not able to find an existing an existing Github Action in the marketplace for HTML linting: https://github.com/marketplace?type=actions&query=html

My path forward would be to create a Python script that utilizes https://pypi.org/project/html-linter/ and add that as a custom Github Action workflow to run this across all HTML files in the repo. It should error if any issues - Info, Warning, and Error - arise and block PRs from being merged. I have not started the investigation into finding a Python CSS linter to do the same with the files in the `css/` folder.

**Comment by caseymhunt** [comment]
@Pepper-Wood I do not believe that contributors outside of the project are able to edit/add Github actions; at least not through the PR mechanism for Hacktoberfest. 

That being said, I do think there is a way to modify the project to take advantage of pre-commit [git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) using [Husky](https://typicode.github.io/husky/#/) which is an npm package- this would increase the complexity of the project. 

As for accessibility (a11y) in HTML, automated accessibility testing is **significantly** more complex than a linter. If you are committed to automated a11y testing, I recommend researching [pa11y](https://github.com/pa11y/pa11y). Otherwise, reliance on developer education and testing will be much more light weight. 

I am willing to work on a pre-commit git hook solution to CSS and HTML linting if there is interest. This solution, also, would allow individual developers to continue to format their files (indents / spaces / etc. ) however they choose, locally. As soon as that code is committed to the git history, it would be automatically formatted using the `lint-staged` plugin configured [as documented](https://github.com/okonet/lint-staged#automatically-fix-code-style-with-prettier-for-javascript-typescript-markdown-html-or-css).

Does this solution sound ok?
